fili-core/src/main/java/com/yahoo/bard/webservice/web/ResponseData.java

```diff
@@ -88,7 +87,8 @@ public Response(
             SimplifiedIntervalList volatileIntervals,
             Map<String, URI> paginationLinks,
             Pagination pagination,
-            ObjectMappersSuite objectMappers
+            ObjectMappersSuite objectMappers,
+            ApiRequest apiRequest
```

Comment:
Basically, anything related to creating JSON or CSV no longer belongs on this class, and should be removed. That includes the `ObjectMappers` and the `jsonFactory` and `csvMapper`.